pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -1326,6 +1328,21 @@ func logChangefeedFailedTelemetry(
 	log.StructuredEvent(ctx, changefeedFailedEvent)
 }
 
+// logChangefeedRetryableErrorTelemetry logs a retryable error for enterprise changefeeds.
+func logChangefeedRetryableErrorTelemetry(ctx context.Context, job *jobs.Job, err error) {
+	var changefeedEventDetails eventpb.CommonChangefeedEventDetails
+	changefeedDetails := job.Details().(jobspb.ChangefeedDetails)
+	changefeedEventDetails = getCommonChangefeedEventDetails(ctx, changefeedDetails, job.Payload().Description)
+
+	changefeedFailedEvent := &eventpb.ChangefeedRetryableError{
+		CommonChangefeedEventDetails: changefeedEventDetails,
+		JobId:                        int64(job.ID()),
```

Comment:
Might be too late since it looks like its already `JobId` for `ChangefeedEmittedBytes` log, but `JobID` would've fit our general patterns more

Reply:
I think I can still change it. We don't have any dashboards consuming it at the moment. I would have to backport this to 22.2 because I backported `CommonChangefeedEventDetails`.